### PR TITLE
Update Emeritus Contributors list

### PR DESCRIPTION
**Pull request by lucasponce**
Hi @xeviknal @abonas!

We are having some activity on the list of contributors and I've updated the list with our Emeritus contributors :) (but we keep welcome any contribution :)).

Please, let me know if that works for you.

Also, this PR has changed in the contacts of the Code of Conduct and some minor re-wording on the contributing doc.

### CODE REVIEW

**Review by jshaughn — DISMISSED** [review]
Good catch, lots of outdated info here.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -51,12 +41,6 @@ Once the issue has been agreed upon and developed, you can send a pull-request.
 The pull-request should have a detailed explanation of the changes that you are doing.
 If you worked on a GitHub issue, please provide the link as part of the description.
 
-If your changes have impact on the front-end, an image showing the changed screen will be
-very helpful to understand your changes. If possible, provide a screenshot before the change
-and another one after the change.
```
Comment (nrfox) [inline CONTRIBUTING.md:56]: This hasn't really been followed but I actually think it would be nice in many cases.

**Review by nrfox — APPROVED** [review]

**Review by leandroberetta — DISMISSED** [review]
LGTM!.

**Review by jmazzitelli — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,37 +12,27 @@ it easier to get your contribution accepted.
...
 on GitHub](https://github.com/kiali/kiali/labels/good%20first%20issue).
 
-### Discussing changes
-
-For large changes it is probably good to first discuss them on the [Kiali-dev](https://groups.google.com/forum/#!forum/kiali-dev) mailing list.
-
 ### Developing
 
-The [README for the server](./README.adoc#building) and the [README for the UI](https://github.com/kiali/kiali-ui#developing) have a pretty exhausting guide on building Kiali server and UI. 
+The [README for the server](./README.adoc#building) and the [README for the UI](./frontend/README.adoc) have a pretty exhausting guide on building Kiali server and UI.
```
Comment (abonas) [inline CONTRIBUTING.md:31]: perhaps "extensive" would sound better than "exhausting" :)

**Review by abonas — DISMISSED** [review]

File: GOVERNANCE.md [hunk 3]
```diff
@@ -145,6 +143,11 @@ If and when contributors commitment levels change, contributors can consider ste
 
 Contact the Maintainers about changing to Emeritus status, or reducing your contributor level.
 
+### Emeritus contributors
+
+* [abonas](https://github.com/abonas)
+* [xeviknal](https://github.com/xeviknal)
+
```
Comment (israel-hdez) [inline GOVERNANCE.md:150]: Minor:

We currently have 14 emeritus contributors (plus Alissa and Xavier who are being moved to this status). Perhaps, worth to mention all 16 emeritus contributors?

**Review by israel-hdez — DISMISSED** [review]
I only have a minor comment. All in all, LGTM.

**Review by Xunzhuo — APPROVED** [review]

File: GOVERNANCE.md (hunk 3 above)
Comment (lucasponce) [reply GOVERNANCE.md:150]: +1 
Totally sense.
I was wondering if doing it in this PR or as a follow-up but I'll update this with the complete list.

**Review by aljesusg — DISMISSED** [review]
LGFM

**Comment by lucasponce** [comment]
Comments addressed.
Sorry, to re-ask a vote, let me know if the latest changes look ok.

Thanks

File: CONTRIBUTING.md (hunk 1 above)
Comment (lucasponce) [reply CONTRIBUTING.md:56]: I've added a comment; yes, it's implicit; we had long templates for PRs with all of this but it created more noise and we saw that people didn't follow it.

**Review by israel-hdez — APPROVED** [review]
All good.

**Pull request merged**